tests/src/test/scala/system/basic/WskBasicTests.scala

```diff
@@ -185,6 +185,27 @@ class WskBasicTests
         stderr should include regex (s"""Unable to get package '$name': The requested resource does not exist. \\(code \\d+\\)""")
     }
 
+    it should "return a list of alphabetized packages" in withAssetCleaner(wskprops) {
```

Comment:
FYI: general functionality/usage tests don't believe in the system package; only tests that are generally associated with system health should be here.

this looks like just a bad merge/need to rebase.